ppr-ui/src/composables/mhrInformation/useMhrUnitNote.ts

```diff
@@ -91,7 +91,7 @@ export const useMhrUnitNote = (unitNoteType: UnitNoteDocTypes) => {
 
     // Unit note expiryDateTime can only be submitted for CAUC, CAUE
     if (![UnitNoteDocTypes.CONTINUED_NOTE_OF_CAUTION,
-      UnitNoteDocTypes.EXTENSION_TO_NOTICE_OF_CAUTION].includes(unitNoteType)) {
+      UnitNoteDocTypes.EXTENSION_TO_NOTICE_OF_CAUTION].includes(unitNoteData.note.documentType)) {
```

Comment:
@dima I believed when we build the payload data, the documentType should be on the unit note. So we don't need to pass it to `useMhrUnitNote` for now. 

I just removed so that I can use `useMhrUnitNote` without providing a document type, but I can change it back if needed.